src/SimpleDbMigrations/ResourceMigration.cs

```diff
@@ -9,6 +9,11 @@ namespace SimpleDbMigrations
 {
     public class ResourceMigration : Migration
     {
+        private static Regex Comments = new Regex(@"--.*$", RegexOptions.Compiled|RegexOptions.Multiline);
+        private static Regex MiddleGo = new Regex(@"\sGO\s", RegexOptions.Compiled|RegexOptions.IgnoreCase);
```

Comment:
The middle go can be troublesome if we would add ^before the  \s it should work.